configure.ac

```diff
@@ -28,10 +28,12 @@ AS_IF([AS_CASE([$host_os],[openedition*],  [false], [true])], [
   CC_CHECK_CFLAGS_APPEND([-pedantic])
 ])
 CC_FLAG_VISIBILITY #[-fvisibility=hidden]
+CC_CHECK_CFLAGS_APPEND([-foobarbaz])
```

Comment:
What does this do?

Reply:
Uh, that's not supposed to be there. I even checked `git show` before pushing the commit... Anyway, removed.